stdlib/public/core/Set.swift

```diff
@@ -390,7 +382,7 @@ extension Set: Collection {
   /// to the set. Don't expect any particular ordering of set elements.
   ///
   /// If the set is empty, the value of this property is `nil`.
-  @inlinable // FIXME(sil-serialize-all)
+  @inlinable
   public var first: Element? {
```

Comment:
I wonder if this really does materially perform better than the default.

Reply:
The default creates an iterator, which complicates the native case, but improves the bridged case -- `_CocoaSet.startIndex` is currently O(n), while the iterator uses fast enumeration. 😦

Ideally this should dispatch to `_variant` and do the right thing in each case. Or we could just remove it.

Reply:
Actually, it turns out this is the correct way to implement this, despite the performance hit. 

Bridged indices are *currently* based on `-[NSDictionary allKeys]`, which is not guaranteed to use the same element ordering as an enumerator. (Which indicates Dictionary/Set iterators may sometimes return elements in a different order than indices. 😭)

I think we should change indices to use enumerators, and soon.